DataFormats/Detectors/Common/CMakeLists.txt

```diff
@@ -16,6 +16,7 @@ o2_add_library(DetectorsCommonDataFormats
                PUBLIC_LINK_LIBRARIES
                ROOT::Core
                ROOT::Geom
+               ROOT::GeomPainter
```

Comment:
just wondering : do we really need to introduce graphics dependencies "so high in the hierarchy" ?

Reply:
Well, I don't really like it either, but it uses TGeoOverlap.h, which belongs to libTGeomPainter.

Reply:
@pzhristov : What shall we do here? I basically don't like the ROOT graphics dependencies, but if we use headers whose typeinfo is in these libs, we get incomplete linking otherwise.

Reply:
I also do not like that for one debug message we have to link one more library. I would propose a separate debug macro that is takin the aligned geometry and dumping the details about the overlaps, while the alignment code simply reports the number of overlaps without the details.